### PR TITLE
Added configuration support

### PR DESCRIPTION
**Pull request by Faldrian**
I added support for including a file.
That file

- can define a context for the templates
- and add helpers / configure arbitrary things with handlebars

:)

### CODE REVIEW

File: src/index.coffee [hunk 1]
```diff
@@ -12,6 +12,11 @@ module.exports = class StaticHandlebarsCompiler
   constructor: (@config) ->
     @outputDirectory = @config?.plugins?.staticHandlebars?.outputDirectory || 'public'
 
+    # Load additional configuration / context from a project-specific file
+    includeFile = @config?.plugins?.staticHandlebars?.includeFile || 'staticHandlebarsExtras.js'
```
Comment (bloudermilk) [inline src/index.coffee]: @Faldrian designating a default filename seems overly prescriptive. Would you mind removing the default here?

File: src/index.coffee [hunk 2]
```diff
@@ -36,12 +41,13 @@ module.exports = class StaticHandlebarsCompiler
     try
       basename = sysPath.basename(path, ".hbs")
       template = handlebars.compile(data)
+      context = if @extras then @extras(handlebars) else {} ;
```
Comment (bloudermilk) [inline src/index.coffee]: @Faldrian no need for the trailing semicolon in CoffeeScript :)

**Comment by bloudermilk** [comment]
@Faldrian this is great! Thank you for taking the time to submit a PR–I'm sure others will find your work extremely helpful. I left a couple comments on the diff with some simple change suggestions. Looking forward to hearing back from you and getting this merged.

File: src/index.coffee (hunk 1 above)
Comment (bloudermilk) [inline src/index.coffee]: @Faldrian worth noting that removing the default also avoid unnecessary synchronous disk access for users who aren't using the includeFile.

**Comment by Faldrian** [comment]
I fixed that, thanks for the review.

**Comment by bloudermilk** [comment]
@Faldrian thanks again for the pull request. Just merged your work in!

**Comment by bloudermilk** [comment]
@Faldrian I made a couple fixes to the README to match your latest commits. If you wouldn't mind it would be great if you could give it a once-over and make sure it's still accurate

**Comment by Faldrian** [comment]
Should be right. I just don't like coffeescript, so the include-file was pure JS at my side. :)

**Pull request merged**